doc/source/data/data.rst

```diff
@@ -0,0 +1,200 @@
+.. include:: /_includes/data/announcement.rst
+
+.. _data:
+
+======================================
+Ray Data: Distributed ML Preprocessing
+======================================
+
+.. _data-intro:
+
+Ray Data is the standard way to load and exchange data in Ray libraries and applications.
+It provides streaming distributed transformations such as maps
+(:meth:`map_batches <ray.data.Datastream.map_batches>`),
+global and grouped aggregations (:class:`GroupedData <ray.data.grouped_data.GroupedData>`), and
+shuffling operations (:meth:`random_shuffle <ray.data.Datastream.random_shuffle>`,
+:meth:`sort <ray.data.Datastream.sort>`,
+:meth:`repartition <ray.data.Datastream.repartition>`),
+and is compatible with a variety of file formats, data sources, and distributed frameworks.
+
+Here's an overview of the integrations with other processing frameworks, file formats, and supported operations,
+as well as a glimpse at the Ray Data API.
+
+Check the :ref:`Input/Output reference <input-output>` to see if your favorite format
+is already supported.
+
+.. image:: images/datastream.svg
+
+..
+  https://docs.google.com/drawings/d/16AwJeBNR46_TsrkOmMbGaBK7u-OPsf_V8fHjU-d2PPQ/edit
+
+-------------------------
+Streaming Batch Inference
+-------------------------
+
+Ray Data simplifies general purpose parallel GPU and CPU compute in Ray through its
+powerful :ref:`Datastream <datastream_concept>` primitive. Datastreams enables workloads such as 
+:ref:`GPU batch inference <ref-use-cases-batch-infer>` efficiently on large datasets. Ray Data manages
```

Comment:
Should we change `large datasets` to `large input data`?

Reply:
I think this is fine. Lower-case datasets just refers to datasets in general, not the Dataset class.